slices/python3.12.yaml

```diff
@@ -0,0 +1,61 @@
+package: python3.12
+
+essential:
+  - python3.12_copyright
+
+slices:
+  # The "core" slice provides a very minimal, yet functioning python3.12.
+  # It includes very few modules from the libpython3.12-stdlib package.
+  core:
+    essential:
+      - libpython3.12-stdlib_core
+      - media-types_data
+      - python3.12-minimal_bins
+
+  # The "standard" slice extends "core" with all the Python
+  # modules from the libpython3.12-stdlib package.
+  standard:
```

Comment:
are we leaving tzdata out on purpose?

Reply:
It actually does come from `libpython3.12-stdlib`. But made it explicit here too: 722ccf3123e24ec09dac6ee8e02ef8690455746d.